src/listener.c

```diff
@@ -49,14 +49,40 @@
 #include "tls.h"
 #include "http.h"
 
-
 static void close_listener(struct ev_loop *, struct Listener *);
 static void accept_cb(struct ev_loop *, struct ev_io *, int);
 static void backoff_timer_cb(struct ev_loop *, struct ev_timer *, int);
 static int init_listener(struct Listener *, const struct Table_head *, struct ev_loop *);
 static void listener_update(struct Listener *, struct Listener *,  const struct Table_head *);
 static void free_listener(struct Listener *);
 
+static const char *boolean_true[] = {
+    "yes",
+    "true",
+    "on",
+};
+
+static const char *boolean_false[] = {
+    "no",
+    "false",
+    "off",
+};
+
+static int
+parse_boolean(char *boolean) {
+    int i;
+    for (i = 0; i < 3; i++) {
+	if (strncasecmp(boolean, boolean_true[i], strlen(boolean)) == 0)
```

Comment:
I think we want to `strcasecmp` here otherwise 'o' will match on since it is matched first and the config tokenizer guarantees the argument is a null terminated string.